Makefile

```diff
@@ -125,7 +125,7 @@ ifneq ("$(NO_CROSS_COMPILER)", "")
 	$(info ************************************************************************************)
 else
 	mkdir -p ./bin/$(TARGETOS)_$(TARGETARCH)
-	$(CC) -o ./bin/$(TARGETOS)_$(TARGETARCH)/syz-executor$(EXE) executor/executor.cc \
+	$(CPP) -o ./bin/$(TARGETOS)_$(TARGETARCH)/syz-executor$(EXE) executor/executor.cc \
```

Comment:
The CC is provided by tools/syz-env. CPP is not provided/not a thing in syzkaller build system.
syz-env takes compiler from CCompiler in sys/targets/targets.go. You need to change it there. That will properly affect how tests build executor, repros built, etc.

Reply:
This means we use a single compiler for C and C++ code? That must be a C++ compiler. When changing in sys/targets/targets.go the C compiler to clang, it complaints that executor.cc is a C++ file, which is correct (it use std::sort).

Reply:
Yep. That's what we do now.
I think OpenBSD already has clang++ there. So it should work.

Reply:
OK. Will try.

Reply:
@dvyukov What is target.CPP in sys/targets/targets.go? No a C++ compiler?

Reply:
https://linux.die.net/man/1/cpp ;)

Reply:
OK. Thanks.